kiwi/system/setup.py

```diff
@@ -322,7 +322,7 @@ def setup_keyboard_map(self) -> None:
                 Path.wipe(self.root_dir + '/etc/vconsole.conf')
                 Command.run([
                     'chroot', self.root_dir, 'systemd-firstboot',
-                    '--keymap=' + self.preferences['keytable']
+                    '--keymap=' + self.preferences['keytable'].replace('.map.gz', '')
```

Comment:
Is it ever possible to be compressed with a different algorithm?

Reply:
I think we have a compatibility problem here. Older versions of systemd-firstboot only accepted the full map name e.g ```us.map.gz```, newer versions only accept the basename e.g ```us```. This and what Neal said is the reason why we pass the map name as it was provided. So in kiwi's XML you can say

```xml
<keytable>us.map.gz</keytable>
```

or

```xml
<keytable>us</keytable>
```

or something else. From that perspective I would not apply the above replacement

Reply:
Thanks both of you for having a look! I was under the impression that it always expected the basename (but it always went through as there was no validation). I will do more tests with older SLE releases and update the PR accordingly